src/App.tsx

```diff
@@ -0,0 +1,74 @@
+import { css, Global } from '@emotion/react';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+
+import Layout from './components/Layout';
+import ColumnDetail from './pages/ColumnDetail';
+import ColumnList from './pages/ColumnList';
+import Home from './pages/Home';
+import Login from './pages/Login';
+import MyAccount from './pages/MyAccount';
+import NotFound from './pages/NotFound';
+import ProductDetail from './pages/ProductDetail';
+import ProductList from './pages/ProductList';
+import { RouterPath } from './shared/path';
+
```

Comment:
`npm run dev` 실행하니까 존재하지 않는 경로라고 에러가 발생하고 있네요. 수정이 필요해보여요.

```suggestion
import { RouterPath } from './utils/path';
```

Reply:
[fix: 절대경로 관련](https://github.com/kakao-tech-campus-2nd-step3/Team2_FE/pull/4/commits/265ea531fc628df0c7ac80c2e783e9f6bd3ac7cf)